details.js

```diff
@@ -0,0 +1,34 @@
+const params = new URLSearchParams(window.location.search);
+const countryName = params.get("country");
+
+let country = {};
```

Comment:
You don't use this variable.
Inside `insertValues` you pass a var named `country`, therefore, line 24 (for example) doesn't access this variable.
it's redundant.